test/libsolidity/semanticTests/errors/named_error_args.sol

```diff
@@ -0,0 +1,10 @@
+error E(uint a, uint b);
+contract C {
+    function f() public pure {
+        revert(E({b: 7, a: 2}));
+    }
+}
+// ====
+// compileViaYul: also
+// ----
+// f() -> FAILURE, hex"85208890", hex"0000000000000000000000000000000000000000000000000000000000000002", hex"0000000000000000000000000000000000000000000000000000000000000007"
```

Comment:
Weird. So the way these numbers get formatted depends not only on the numbers themselves but also on slight variations in the code in the test case? They're just ints in `simple.sol` below.

Reply:
It depends on whether or not I changed the formatting after they were auto-generated...